Code/Framework/AzFramework/AzFramework/Entity/EntityDebugDisplayBus.h

```diff
@@ -48,7 +48,12 @@ namespace AzFramework
         virtual void SetColor(const AZ::Vector4& color) { (void)color; }
         virtual void SetAlpha(float a) { (void)a; }
         virtual void DrawQuad(const AZ::Vector3& p1, const AZ::Vector3& p2, const AZ::Vector3& p3, const AZ::Vector3& p4) { (void)p1; (void)p2; (void)p3; (void)p4; }
-        virtual void DrawQuad(float width, float height) { (void)width; (void)height; }
+        virtual void DrawQuad(float width, float height, bool drawShaded)
+        {
+            (void)width;
+            (void)height;
+            (void)drawShaded;
+        }
```

Comment:
Lets make these match the existing style in the file.

Also can we add a default true value for drawShaded that matches the previous behavior.